schemas/2023/08/bundled/outputs/resource/list.json

```diff
@@ -73,8 +73,8 @@
       }
     },
     "requires": {
-      "title": "Required DSC Resource Provider",
-      "description": "Defines the fully qualified type name of the DSC Resource Provider the DSC Resource depends on.",
+      "title": "Required DSC Resource Adapter",
```

Comment:
We should not change the existing schemas, but update the `$schema` in the sourc config, rename the file and update the strings in source, and generate the schemas for the new canonical URI - otherwise these schemas can't be used with the previous release. This change is breaking, so needs a new URI.

Reply:
reverting entire `schemas` and `docs` folders